.github/workflows/continous_integration.yml

```diff
@@ -108,5 +102,5 @@ jobs:
         uses: coverallsapp/github-action@v2
         with:
           parallel-finished: true
-          carryforward: "python3.9,python3.10,python3.11"
+          carryforward: "python3.10,python3.11,python3.12"
```

Comment:
Should we not also build for 3.9 if we test it?

Reply:
Done!